canonical/vanilla-framework
# Adjust p-nudge to better align text to the baseline grid

## Done

* changed all text nudges that previously had a nudge of `0.4rem` to `0.375rem` to align default text (and some headings) better to the baseline grid
* I tried to test most of the things that depend on p-nudge
  * While doing that I found a couple of misalignments to the baseline grid (that also existed for the previous nudges)
* One small drive-by typo fix 

## QA

- Check out all the components that depend on p-nudge and see if they align properly to the baseline grid.
   - eg. default text, some headings, input fields, buttons etc.

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots

### Before
![Screenshot 2024-01-24 at 11 44 57](https://github.com/canonical/vanilla-framework/assets/68336485/fa23c034-0438-481a-a322-eb270e0138a7)

### After
![Screenshot 2024-01-24 at 11 44 26](https://github.com/canonical/vanilla-framework/assets/68336485/2ca66982-5dd2-4e23-8fad-f44e4b76a7f9)
